### DISCUSSION TITLE
Script "rssi=Wifi#RSSI" will result in: var not found: Wifi#RSSI

### DISCUSSION THREAD

**Discussion by gneiss15** [arendst/Tasmota#19720] [Q&A] [answered]
I am just starting with Tasmota, but had manged so far to build my own FW whit support for "Smart Meter" and "Wifi Kit 32" and its integrated Display.

I now have trobble with my WLAN (RSSI to low) at the place the Kit should be installed (far away from Router, as usual for Smartmeters).
ThereforeI try to get the current RSSI displayed on the OLED.
I wrote the following simple script (just for debug):
#######
>D
rssi=0
cnt=0
>S
;rssi=Wifi#RSSI

;print %cnt%
if cnt==0
then
; print cnt == 0
=>SetOption56 1
=>SetOption57 1
=>DisplayMode 0
=>DisplayDimmer 100
=>DisplayText [s1l3c1]RSSI: %rssi%
cnt+=1
endif
#######
Most if this is taken from "example" pages for Tasmota scripting.
But when I enable the script I get:
var not found: Wifi#RSSI
inside the Console.
The text was correctly displayed

What I am doing wrong ?
Did I miss an option that has to be defined ?

BTW: I hardly found out, that the aboce commands (SetOption.. and Display..) only works when they are:
- start at the 1. Col
- - Have "=>" in front
- So that's not clear to why that is the case (inside the docs that isn't explained).

**Comment by sfromis** [comment]
`>S` is not the right section for doing JSON references.

**Comment by gneiss15** [comment, marked as answer]
First, thanks for Yor answer..

Makes sense...

Inside the example at https://tasmota.github.io/docs/Scripting-Language/#scripting-language-example it's inside >T
So I will try to put it there.....

And Yes, this works :-)
